docs/install.md

```diff
@@ -1,12 +1,15 @@
 # Installation
 
 ## Prerequisites
+
+* Kubernetes Version >= 1.20
+  * Versions 1.18 - 1.19 may require the `VolumeSnapshotDataSource` feature gate be enabled on `kubelet`. 
```

Comment:
1.18 and 1.19 are long EOL and not even supported by EKS anymore, maybe we can finally just drop the extra note?

Reply:
I agree in spirit, but you can still create new 1.19 clusters on EKS. 

https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html

Reply:
![Screenshot_20220916-160653](https://user-images.githubusercontent.com/6501170/190724544-51933463-0f7d-4b61-b166-a5cbc7e25f88.png)

When I go to create a cluster in the console, it doesn't give 1.19 as an option. Is it possible those docs are out of date? Or maybe it's possible only via the CLI/API for some reason?

Reply:
the docs are out of date...... so yes now i'm fine removing it.